index.djs

```diff
@@ -0,0 +1,75 @@
+such setStyle much elementID property value
+    very element is plz dogeument.getElementById with elementID
+    plz element.style.setProperty with property value
+wow
+
+such hideLoadingScreen
+    plz setStyle with "loadingElement" "display" "none"
+wow
+
+plz $ with much
+wow& hideLoadingScreen()
+
+very animations = [/*
+*/    "spin",
```

Comment:
The comment is needed because dogescript adds a semicolon to the end of every line starting with "very". I'll change this to statements constructing the array instead though e.g `animation dose push "zoom"` ...